src/common/mailer/mailer.manager.ts

```diff
@@ -1,20 +1,35 @@
 import { MailerService } from '@nestjs-modules/mailer';
-import { Injectable } from '@nestjs/common';
```

Comment:
This is the Mailer Manager, don't put the Substrate service code in here. Separate it somewhere else please.

Reply:
Don't inject an `ApiPromise` inside the `mailer.manager.ts`. This file is specifically made for mail not Substrate.